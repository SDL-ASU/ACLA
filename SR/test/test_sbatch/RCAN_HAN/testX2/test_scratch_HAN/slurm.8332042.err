  0%|                                                     | 0/5 [00:00<?, ?it/s] 20%|█████████                                    | 1/5 [00:00<00:01,  2.07it/s] 40%|██████████████████                           | 2/5 [00:00<00:01,  2.59it/s] 60%|███████████████████████████                  | 3/5 [00:01<00:00,  2.84it/s] 80%|████████████████████████████████████         | 4/5 [00:01<00:00,  2.94it/s]100%|█████████████████████████████████████████████| 5/5 [00:01<00:00,  2.86it/s]
  0%|                                                    | 0/14 [00:00<?, ?it/s]  7%|███▏                                        | 1/14 [00:00<00:09,  1.42it/s] 14%|██████▎                                     | 2/14 [00:01<00:06,  1.86it/s] 21%|█████████▍                                  | 3/14 [00:01<00:05,  2.15it/s] 29%|████████████▌                               | 4/14 [00:01<00:04,  2.01it/s] 36%|███████████████▋                            | 5/14 [00:02<00:04,  2.08it/s] 43%|██████████████████▊                         | 6/14 [00:02<00:03,  2.13it/s] 50%|██████████████████████                      | 7/14 [00:03<00:03,  2.25it/s] 57%|█████████████████████████▏                  | 8/14 [00:03<00:02,  2.35it/s] 64%|████████████████████████████▎               | 9/14 [00:03<00:02,  2.39it/s] 71%|██████████████████████████████▋            | 10/14 [00:04<00:01,  2.47it/s] 79%|█████████████████████████████████▊         | 11/14 [00:04<00:01,  2.50it/s] 86%|████████████████████████████████████▊      | 12/14 [00:04<00:00,  2.48it/s] 93%|███████████████████████████████████████▉   | 13/14 [00:05<00:00,  2.50it/s]100%|███████████████████████████████████████████| 14/14 [00:05<00:00,  2.54it/s]
  0%|                                                   | 0/100 [00:00<?, ?it/s]  1%|▍                                          | 1/100 [00:00<00:42,  2.35it/s]  2%|▊                                          | 2/100 [00:00<00:36,  2.67it/s]  3%|█▎                                         | 3/100 [00:01<00:34,  2.77it/s]  4%|█▋                                         | 4/100 [00:01<00:36,  2.67it/s]  5%|██▏                                        | 5/100 [00:01<00:34,  2.76it/s]  6%|██▌                                        | 6/100 [00:02<00:33,  2.80it/s]  7%|███                                        | 7/100 [00:02<00:32,  2.85it/s]  8%|███▍                                       | 8/100 [00:02<00:31,  2.88it/s]  9%|███▊                                       | 9/100 [00:03<00:31,  2.88it/s] 10%|████▏                                     | 10/100 [00:03<00:31,  2.85it/s] 11%|████▌                                     | 11/100 [00:03<00:31,  2.80it/s] 12%|█████                                     | 12/100 [00:04<00:31,  2.80it/s] 13%|█████▍                                    | 13/100 [00:04<00:30,  2.82it/s] 14%|█████▉                                    | 14/100 [00:04<00:30,  2.84it/s] 15%|██████▎                                   | 15/100 [00:05<00:29,  2.85it/s] 16%|██████▋                                   | 16/100 [00:05<00:29,  2.85it/s] 17%|███████▏                                  | 17/100 [00:05<00:29,  2.86it/s] 18%|███████▌                                  | 18/100 [00:06<00:28,  2.86it/s] 19%|███████▉                                  | 19/100 [00:06<00:29,  2.78it/s] 20%|████████▍                                 | 20/100 [00:07<00:28,  2.79it/s] 21%|████████▊                                 | 21/100 [00:07<00:28,  2.76it/s] 22%|█████████▏                                | 22/100 [00:07<00:28,  2.78it/s] 23%|█████████▋                                | 23/100 [00:08<00:27,  2.79it/s] 24%|██████████                                | 24/100 [00:08<00:27,  2.79it/s] 25%|██████████▌                               | 25/100 [00:08<00:26,  2.80it/s] 26%|██████████▉                               | 26/100 [00:09<00:26,  2.81it/s] 27%|███████████▎                              | 27/100 [00:09<00:25,  2.81it/s] 28%|███████████▊                              | 28/100 [00:09<00:25,  2.83it/s] 29%|████████████▏                             | 29/100 [00:10<00:25,  2.84it/s] 30%|████████████▌                             | 30/100 [00:10<00:24,  2.85it/s] 31%|█████████████                             | 31/100 [00:10<00:24,  2.86it/s] 32%|█████████████▍                            | 32/100 [00:11<00:23,  2.87it/s] 33%|█████████████▊                            | 33/100 [00:11<00:23,  2.86it/s] 34%|██████████████▎                           | 34/100 [00:11<00:23,  2.87it/s] 35%|██████████████▋                           | 35/100 [00:12<00:22,  2.87it/s] 36%|███████████████                           | 36/100 [00:12<00:22,  2.86it/s] 37%|███████████████▌                          | 37/100 [00:12<00:22,  2.86it/s] 38%|███████████████▉                          | 38/100 [00:13<00:21,  2.87it/s] 39%|████████████████▍                         | 39/100 [00:13<00:21,  2.88it/s] 40%|████████████████▊                         | 40/100 [00:13<00:20,  2.89it/s] 41%|█████████████████▏                        | 41/100 [00:14<00:20,  2.87it/s] 42%|█████████████████▋                        | 42/100 [00:14<00:20,  2.87it/s] 43%|██████████████████                        | 43/100 [00:15<00:19,  2.87it/s] 44%|██████████████████▍                       | 44/100 [00:15<00:19,  2.87it/s] 45%|██████████████████▉                       | 45/100 [00:15<00:19,  2.87it/s] 46%|███████████████████▎                      | 46/100 [00:16<00:18,  2.87it/s] 47%|███████████████████▋                      | 47/100 [00:16<00:18,  2.87it/s] 48%|████████████████████▏                     | 48/100 [00:16<00:18,  2.87it/s] 49%|████████████████████▌                     | 49/100 [00:17<00:17,  2.87it/s] 50%|█████████████████████                     | 50/100 [00:17<00:17,  2.86it/s] 51%|█████████████████████▍                    | 51/100 [00:17<00:17,  2.86it/s] 52%|█████████████████████▊                    | 52/100 [00:18<00:16,  2.87it/s] 53%|██████████████████████▎                   | 53/100 [00:18<00:16,  2.87it/s] 54%|██████████████████████▋                   | 54/100 [00:18<00:16,  2.87it/s] 55%|███████████████████████                   | 55/100 [00:19<00:15,  2.87it/s] 56%|███████████████████████▌                  | 56/100 [00:19<00:15,  2.87it/s] 57%|███████████████████████▉                  | 57/100 [00:19<00:15,  2.86it/s] 58%|████████████████████████▎                 | 58/100 [00:20<00:14,  2.87it/s] 59%|████████████████████████▊                 | 59/100 [00:20<00:14,  2.87it/s] 60%|█████████████████████████▏                | 60/100 [00:20<00:13,  2.87it/s] 61%|█████████████████████████▌                | 61/100 [00:21<00:13,  2.87it/s] 62%|██████████████████████████                | 62/100 [00:21<00:13,  2.88it/s] 63%|██████████████████████████▍               | 63/100 [00:21<00:12,  2.87it/s] 64%|██████████████████████████▉               | 64/100 [00:22<00:12,  2.87it/s] 65%|███████████████████████████▎              | 65/100 [00:22<00:12,  2.87it/s] 66%|███████████████████████████▋              | 66/100 [00:22<00:11,  2.88it/s] 67%|████████████████████████████▏             | 67/100 [00:23<00:11,  2.88it/s] 68%|████████████████████████████▌             | 68/100 [00:23<00:11,  2.88it/s] 69%|████████████████████████████▉             | 69/100 [00:23<00:10,  2.89it/s] 70%|█████████████████████████████▍            | 70/100 [00:24<00:10,  2.89it/s] 71%|█████████████████████████████▊            | 71/100 [00:24<00:10,  2.89it/s] 72%|██████████████████████████████▏           | 72/100 [00:24<00:09,  2.88it/s] 73%|██████████████████████████████▋           | 73/100 [00:25<00:09,  2.89it/s] 74%|███████████████████████████████           | 74/100 [00:25<00:09,  2.88it/s] 75%|███████████████████████████████▌          | 75/100 [00:26<00:08,  2.88it/s] 76%|███████████████████████████████▉          | 76/100 [00:26<00:08,  2.89it/s] 77%|████████████████████████████████▎         | 77/100 [00:26<00:07,  2.89it/s] 78%|████████████████████████████████▊         | 78/100 [00:26<00:07,  2.90it/s] 79%|█████████████████████████████████▏        | 79/100 [00:27<00:07,  2.90it/s] 80%|█████████████████████████████████▌        | 80/100 [00:27<00:06,  2.90it/s] 81%|██████████████████████████████████        | 81/100 [00:27<00:06,  2.90it/s] 82%|██████████████████████████████████▍       | 82/100 [00:28<00:06,  2.91it/s] 83%|██████████████████████████████████▊       | 83/100 [00:28<00:05,  2.90it/s] 84%|███████████████████████████████████▎      | 84/100 [00:28<00:05,  2.90it/s] 85%|███████████████████████████████████▋      | 85/100 [00:29<00:05,  2.90it/s] 86%|████████████████████████████████████      | 86/100 [00:29<00:04,  2.90it/s] 87%|████████████████████████████████████▌     | 87/100 [00:29<00:04,  2.90it/s] 88%|████████████████████████████████████▉     | 88/100 [00:30<00:04,  2.90it/s] 89%|█████████████████████████████████████▍    | 89/100 [00:30<00:03,  2.90it/s] 90%|█████████████████████████████████████▊    | 90/100 [00:31<00:03,  2.90it/s] 91%|██████████████████████████████████████▏   | 91/100 [00:31<00:03,  2.90it/s] 92%|██████████████████████████████████████▋   | 92/100 [00:31<00:02,  2.90it/s] 93%|███████████████████████████████████████   | 93/100 [00:32<00:02,  2.91it/s] 94%|███████████████████████████████████████▍  | 94/100 [00:32<00:02,  2.91it/s] 95%|███████████████████████████████████████▉  | 95/100 [00:32<00:01,  2.91it/s] 96%|████████████████████████████████████████▎ | 96/100 [00:32<00:01,  2.92it/s] 97%|████████████████████████████████████████▋ | 97/100 [00:33<00:01,  2.92it/s] 98%|█████████████████████████████████████████▏| 98/100 [00:33<00:00,  2.92it/s] 99%|█████████████████████████████████████████▌| 99/100 [00:33<00:00,  2.92it/s]100%|█████████████████████████████████████████| 100/100 [00:34<00:00,  2.92it/s]
  0%|                                                   | 0/100 [00:00<?, ?it/s]  1%|▍                                          | 1/100 [00:01<01:48,  1.10s/it]  2%|▊                                          | 2/100 [00:02<02:05,  1.28s/it]  3%|█▎                                         | 3/100 [00:03<01:53,  1.17s/it]  4%|█▋                                         | 4/100 [00:04<01:50,  1.15s/it]  5%|██▏                                        | 5/100 [00:06<01:54,  1.20s/it]  6%|██▌                                        | 6/100 [00:07<01:51,  1.18s/it]  7%|███                                        | 7/100 [00:08<01:54,  1.23s/it]  8%|███▍                                       | 8/100 [00:09<01:50,  1.21s/it]  9%|███▊                                       | 9/100 [00:10<01:47,  1.18s/it] 10%|████▏                                     | 10/100 [00:11<01:47,  1.20s/it] 11%|████▌                                     | 11/100 [00:12<01:45,  1.18s/it] 12%|█████                                     | 12/100 [00:13<01:42,  1.17s/it] 13%|█████▍                                    | 13/100 [00:15<01:40,  1.16s/it] 14%|█████▉                                    | 14/100 [00:16<01:38,  1.14s/it] 15%|██████▎                                   | 15/100 [00:16<01:35,  1.13s/it] 16%|██████▋                                   | 16/100 [00:17<01:34,  1.12s/it] 17%|███████▏                                  | 17/100 [00:19<01:33,  1.12s/it] 18%|███████▌                                  | 18/100 [00:20<01:31,  1.12s/it] 19%|███████▉                                  | 19/100 [00:21<01:30,  1.11s/it] 20%|████████▍                                 | 20/100 [00:22<01:28,  1.10s/it] 21%|████████▊                                 | 21/100 [00:23<01:27,  1.11s/it] 22%|█████████▏                                | 22/100 [00:24<01:26,  1.11s/it] 23%|█████████▋                                | 23/100 [00:25<01:25,  1.11s/it] 24%|██████████                                | 24/100 [00:26<01:23,  1.10s/it] 25%|██████████▌                               | 25/100 [00:27<01:23,  1.11s/it] 26%|██████████▉                               | 26/100 [00:29<01:23,  1.13s/it] 27%|███████████▎                              | 27/100 [00:30<01:21,  1.12s/it] 28%|███████████▊                              | 28/100 [00:31<01:20,  1.12s/it] 29%|████████████▏                             | 29/100 [00:32<01:19,  1.12s/it] 30%|████████████▌                             | 30/100 [00:33<01:17,  1.11s/it] 31%|█████████████                             | 31/100 [00:34<01:16,  1.11s/it] 32%|█████████████▍                            | 32/100 [00:35<01:15,  1.11s/it] 33%|█████████████▊                            | 33/100 [00:36<01:14,  1.11s/it] 34%|██████████████▎                           | 34/100 [00:37<01:12,  1.10s/it] 35%|██████████████▋                           | 35/100 [00:38<01:11,  1.10s/it] 36%|███████████████                           | 36/100 [00:39<01:10,  1.10s/it] 37%|███████████████▌                          | 37/100 [00:40<01:08,  1.09s/it] 38%|███████████████▉                          | 38/100 [00:41<01:07,  1.09s/it] 39%|████████████████▍                         | 39/100 [00:42<01:05,  1.08s/it] 40%|████████████████▊                         | 40/100 [00:43<01:04,  1.08s/it] 41%|█████████████████▏                        | 41/100 [00:44<01:04,  1.09s/it] 42%|█████████████████▋                        | 42/100 [00:45<01:02,  1.08s/it] 43%|██████████████████                        | 43/100 [00:46<01:01,  1.08s/it] 44%|██████████████████▍                       | 44/100 [00:47<01:00,  1.08s/it] 45%|██████████████████▉                       | 45/100 [00:48<00:59,  1.08s/it] 46%|███████████████████▎                      | 46/100 [00:49<00:57,  1.07s/it] 47%|███████████████████▋                      | 47/100 [00:50<00:56,  1.07s/it] 48%|████████████████████▏                     | 48/100 [00:51<00:55,  1.07s/it] 49%|████████████████████▌                     | 49/100 [00:52<00:54,  1.07s/it] 50%|█████████████████████                     | 50/100 [00:53<00:53,  1.07s/it] 51%|█████████████████████▍                    | 51/100 [00:54<00:52,  1.07s/it] 52%|█████████████████████▊                    | 52/100 [00:55<00:51,  1.07s/it] 53%|██████████████████████▎                   | 53/100 [00:56<00:50,  1.06s/it] 54%|██████████████████████▋                   | 54/100 [00:57<00:49,  1.07s/it] 55%|███████████████████████                   | 55/100 [00:58<00:48,  1.07s/it] 56%|███████████████████████▌                  | 56/100 [00:59<00:46,  1.06s/it] 57%|███████████████████████▉                  | 57/100 [01:00<00:45,  1.07s/it] 58%|████████████████████████▎                 | 58/100 [01:01<00:44,  1.06s/it] 59%|████████████████████████▊                 | 59/100 [01:02<00:43,  1.06s/it] 60%|█████████████████████████▏                | 60/100 [01:03<00:42,  1.06s/it] 61%|█████████████████████████▌                | 61/100 [01:04<00:41,  1.06s/it] 62%|██████████████████████████                | 62/100 [01:05<00:40,  1.06s/it] 63%|██████████████████████████▍               | 63/100 [01:06<00:39,  1.06s/it] 64%|██████████████████████████▉               | 64/100 [01:07<00:37,  1.05s/it] 65%|███████████████████████████▎              | 65/100 [01:08<00:36,  1.05s/it] 66%|███████████████████████████▋              | 66/100 [01:09<00:35,  1.06s/it] 67%|████████████████████████████▏             | 67/100 [01:10<00:34,  1.06s/it] 68%|████████████████████████████▌             | 68/100 [01:11<00:33,  1.05s/it] 69%|████████████████████████████▉             | 69/100 [01:12<00:32,  1.05s/it] 70%|█████████████████████████████▍            | 70/100 [01:13<00:31,  1.05s/it] 71%|█████████████████████████████▊            | 71/100 [01:14<00:30,  1.05s/it] 72%|██████████████████████████████▏           | 72/100 [01:15<00:29,  1.06s/it] 73%|██████████████████████████████▋           | 73/100 [01:16<00:28,  1.05s/it] 74%|███████████████████████████████           | 74/100 [01:17<00:27,  1.05s/it] 75%|███████████████████████████████▌          | 75/100 [01:18<00:26,  1.05s/it] 76%|███████████████████████████████▉          | 76/100 [01:19<00:25,  1.05s/it] 77%|████████████████████████████████▎         | 77/100 [01:20<00:24,  1.05s/it] 78%|████████████████████████████████▊         | 78/100 [01:21<00:23,  1.05s/it] 79%|█████████████████████████████████▏        | 79/100 [01:22<00:21,  1.05s/it] 80%|█████████████████████████████████▌        | 80/100 [01:23<00:20,  1.04s/it] 81%|██████████████████████████████████        | 81/100 [01:24<00:19,  1.04s/it] 82%|██████████████████████████████████▍       | 82/100 [01:25<00:18,  1.04s/it] 83%|██████████████████████████████████▊       | 83/100 [01:26<00:17,  1.04s/it] 84%|███████████████████████████████████▎      | 84/100 [01:27<00:16,  1.04s/it] 85%|███████████████████████████████████▋      | 85/100 [01:28<00:15,  1.05s/it] 86%|████████████████████████████████████      | 86/100 [01:30<00:14,  1.05s/it] 87%|████████████████████████████████████▌     | 87/100 [01:31<00:13,  1.05s/it] 88%|████████████████████████████████████▉     | 88/100 [01:32<00:12,  1.05s/it] 89%|█████████████████████████████████████▍    | 89/100 [01:33<00:11,  1.05s/it] 90%|█████████████████████████████████████▊    | 90/100 [01:34<00:10,  1.05s/it] 91%|██████████████████████████████████████▏   | 91/100 [01:35<00:09,  1.05s/it] 92%|██████████████████████████████████████▋   | 92/100 [01:36<00:08,  1.05s/it] 93%|███████████████████████████████████████   | 93/100 [01:37<00:07,  1.05s/it] 94%|███████████████████████████████████████▍  | 94/100 [01:38<00:06,  1.05s/it] 95%|███████████████████████████████████████▉  | 95/100 [01:40<00:05,  1.06s/it] 96%|████████████████████████████████████████▎ | 96/100 [01:41<00:04,  1.05s/it] 97%|████████████████████████████████████████▋ | 97/100 [01:42<00:03,  1.05s/it] 98%|█████████████████████████████████████████▏| 98/100 [01:43<00:02,  1.05s/it] 99%|█████████████████████████████████████████▌| 99/100 [01:44<00:01,  1.05s/it]100%|█████████████████████████████████████████| 100/100 [01:45<00:00,  1.05s/it]
  0%|                                                   | 0/109 [00:00<?, ?it/s]  1%|▍                                          | 1/109 [00:01<02:38,  1.47s/it]  2%|▊                                          | 2/109 [00:02<02:26,  1.37s/it]  3%|█▏                                         | 3/109 [00:04<02:21,  1.34s/it]  4%|█▌                                         | 4/109 [00:05<02:20,  1.34s/it]  5%|█▉                                         | 5/109 [00:06<02:17,  1.33s/it]  6%|██▎                                        | 6/109 [00:07<02:14,  1.31s/it]  6%|██▊                                        | 7/109 [00:09<02:13,  1.30s/it]  7%|███▏                                       | 8/109 [00:10<02:12,  1.31s/it]  8%|███▌                                       | 9/109 [00:11<02:09,  1.29s/it]  9%|███▊                                      | 10/109 [00:12<02:08,  1.30s/it] 10%|████▏                                     | 11/109 [00:14<02:07,  1.30s/it] 11%|████▌                                     | 12/109 [00:15<02:05,  1.30s/it] 12%|█████                                     | 13/109 [00:16<02:03,  1.29s/it] 13%|█████▍                                    | 14/109 [00:18<02:02,  1.29s/it] 14%|█████▊                                    | 15/109 [00:19<01:59,  1.28s/it] 15%|██████▏                                   | 16/109 [00:20<01:57,  1.27s/it] 16%|██████▌                                   | 17/109 [00:21<01:56,  1.27s/it] 17%|██████▉                                   | 18/109 [00:22<01:55,  1.27s/it] 17%|███████▎                                  | 19/109 [00:24<01:54,  1.28s/it] 18%|███████▋                                  | 20/109 [00:25<01:52,  1.27s/it] 19%|████████                                  | 21/109 [00:26<01:51,  1.27s/it] 20%|████████▍                                 | 22/109 [00:27<01:50,  1.27s/it] 21%|████████▊                                 | 23/109 [00:29<01:49,  1.28s/it] 22%|█████████▏                                | 24/109 [00:30<01:48,  1.27s/it] 23%|█████████▋                                | 25/109 [00:31<01:46,  1.27s/it] 24%|██████████                                | 26/109 [00:33<01:45,  1.27s/it] 25%|██████████▍                               | 27/109 [00:34<01:44,  1.27s/it] 26%|██████████▊                               | 28/109 [00:35<01:43,  1.28s/it] 27%|███████████▏                              | 29/109 [00:37<01:42,  1.28s/it] 28%|███████████▌                              | 30/109 [00:38<01:40,  1.28s/it] 28%|███████████▉                              | 31/109 [00:39<01:39,  1.28s/it] 29%|████████████▎                             | 32/109 [00:40<01:38,  1.27s/it] 30%|████████████▋                             | 33/109 [00:42<01:37,  1.28s/it] 31%|█████████████                             | 34/109 [00:43<01:36,  1.28s/it] 32%|█████████████▍                            | 35/109 [00:44<01:35,  1.28s/it] 33%|█████████████▊                            | 36/109 [00:46<01:33,  1.28s/it] 34%|██████████████▎                           | 37/109 [00:47<01:32,  1.28s/it] 35%|██████████████▋                           | 38/109 [00:48<01:31,  1.28s/it] 36%|███████████████                           | 39/109 [00:50<01:29,  1.28s/it] 37%|███████████████▍                          | 40/109 [00:51<01:28,  1.28s/it] 38%|███████████████▊                          | 41/109 [00:52<01:27,  1.28s/it] 39%|████████████████▏                         | 42/109 [00:53<01:26,  1.28s/it] 39%|████████████████▌                         | 43/109 [00:55<01:24,  1.29s/it] 40%|████████████████▉                         | 44/109 [00:56<01:23,  1.29s/it] 41%|█████████████████▎                        | 45/109 [00:57<01:22,  1.28s/it] 42%|█████████████████▋                        | 46/109 [00:59<01:21,  1.29s/it] 43%|██████████████████                        | 47/109 [01:00<01:19,  1.29s/it] 44%|██████████████████▍                       | 48/109 [01:01<01:18,  1.29s/it] 45%|██████████████████▉                       | 49/109 [01:03<01:17,  1.29s/it] 46%|███████████████████▎                      | 50/109 [01:04<01:16,  1.29s/it] 47%|███████████████████▋                      | 51/109 [01:05<01:15,  1.29s/it] 48%|████████████████████                      | 52/109 [01:07<01:13,  1.29s/it] 49%|████████████████████▍                     | 53/109 [01:08<01:12,  1.29s/it] 50%|████████████████████▊                     | 54/109 [01:09<01:11,  1.29s/it] 50%|█████████████████████▏                    | 55/109 [01:11<01:09,  1.29s/it] 51%|█████████████████████▌                    | 56/109 [01:12<01:08,  1.29s/it] 52%|█████████████████████▉                    | 57/109 [01:13<01:07,  1.29s/it] 53%|██████████████████████▎                   | 58/109 [01:14<01:05,  1.29s/it] 54%|██████████████████████▋                   | 59/109 [01:15<01:04,  1.29s/it] 55%|███████████████████████                   | 60/109 [01:17<01:03,  1.29s/it] 56%|███████████████████████▌                  | 61/109 [01:18<01:01,  1.29s/it] 57%|███████████████████████▉                  | 62/109 [01:19<01:00,  1.29s/it] 58%|████████████████████████▎                 | 63/109 [01:20<00:59,  1.28s/it] 59%|████████████████████████▋                 | 64/109 [01:22<00:57,  1.28s/it] 60%|█████████████████████████                 | 65/109 [01:23<00:56,  1.28s/it] 61%|█████████████████████████▍                | 66/109 [01:24<00:55,  1.29s/it] 61%|█████████████████████████▊                | 67/109 [01:26<00:54,  1.29s/it] 62%|██████████████████████████▏               | 68/109 [01:27<00:52,  1.29s/it] 63%|██████████████████████████▌               | 69/109 [01:28<00:51,  1.29s/it] 64%|██████████████████████████▉               | 70/109 [01:30<00:50,  1.29s/it] 65%|███████████████████████████▎              | 71/109 [01:31<00:48,  1.29s/it] 66%|███████████████████████████▋              | 72/109 [01:32<00:47,  1.29s/it] 67%|████████████████████████████▏             | 73/109 [01:33<00:46,  1.29s/it] 68%|████████████████████████████▌             | 74/109 [01:35<00:44,  1.29s/it] 69%|████████████████████████████▉             | 75/109 [01:36<00:43,  1.28s/it] 70%|█████████████████████████████▎            | 76/109 [01:37<00:42,  1.28s/it] 71%|█████████████████████████████▋            | 77/109 [01:38<00:41,  1.28s/it] 72%|██████████████████████████████            | 78/109 [01:40<00:39,  1.28s/it] 72%|██████████████████████████████▍           | 79/109 [01:41<00:38,  1.28s/it] 73%|██████████████████████████████▊           | 80/109 [01:42<00:37,  1.29s/it] 74%|███████████████████████████████▏          | 81/109 [01:44<00:35,  1.29s/it] 75%|███████████████████████████████▌          | 82/109 [01:45<00:34,  1.29s/it] 76%|███████████████████████████████▉          | 83/109 [01:46<00:33,  1.29s/it] 77%|████████████████████████████████▎         | 84/109 [01:48<00:32,  1.29s/it] 78%|████████████████████████████████▊         | 85/109 [01:49<00:30,  1.29s/it] 79%|█████████████████████████████████▏        | 86/109 [01:50<00:29,  1.29s/it] 80%|█████████████████████████████████▌        | 87/109 [01:51<00:28,  1.29s/it] 81%|█████████████████████████████████▉        | 88/109 [01:53<00:27,  1.29s/it] 82%|██████████████████████████████████▎       | 89/109 [01:54<00:25,  1.29s/it] 83%|██████████████████████████████████▋       | 90/109 [01:55<00:24,  1.29s/it] 83%|███████████████████████████████████       | 91/109 [01:57<00:23,  1.29s/it] 84%|███████████████████████████████████▍      | 92/109 [01:58<00:21,  1.28s/it] 85%|███████████████████████████████████▊      | 93/109 [01:59<00:20,  1.28s/it] 86%|████████████████████████████████████▏     | 94/109 [02:00<00:19,  1.28s/it] 87%|████████████████████████████████████▌     | 95/109 [02:01<00:17,  1.28s/it] 88%|████████████████████████████████████▉     | 96/109 [02:03<00:16,  1.28s/it] 89%|█████████████████████████████████████▍    | 97/109 [02:04<00:15,  1.28s/it] 90%|█████████████████████████████████████▊    | 98/109 [02:05<00:14,  1.28s/it] 91%|██████████████████████████████████████▏   | 99/109 [02:06<00:12,  1.28s/it] 92%|█████████████████████████████████████▌   | 100/109 [02:08<00:11,  1.28s/it] 93%|█████████████████████████████████████▉   | 101/109 [02:09<00:10,  1.28s/it] 94%|██████████████████████████████████████▎  | 102/109 [02:10<00:08,  1.28s/it] 94%|██████████████████████████████████████▋  | 103/109 [02:11<00:07,  1.28s/it] 95%|███████████████████████████████████████  | 104/109 [02:13<00:06,  1.28s/it] 96%|███████████████████████████████████████▍ | 105/109 [02:14<00:05,  1.28s/it] 97%|███████████████████████████████████████▊ | 106/109 [02:15<00:03,  1.28s/it] 98%|████████████████████████████████████████▏| 107/109 [02:16<00:02,  1.28s/it] 99%|████████████████████████████████████████▌| 108/109 [02:18<00:01,  1.28s/it]100%|█████████████████████████████████████████| 109/109 [02:19<00:00,  1.28s/it]
