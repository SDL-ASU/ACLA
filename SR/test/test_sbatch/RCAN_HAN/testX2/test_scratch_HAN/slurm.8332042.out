Making model...
Loading model from /data/yyang409/yancheng/SR_baseline_results/RCAN_HAN/x2/RCAN_HAN_x2_scratch_gammaL1C1_fixed/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 1.78s, ave time: 0.36s

Making model...
Loading model from /data/yyang409/yancheng/SR_baseline_results/RCAN_HAN/x2/RCAN_HAN_x2_scratch_gammaL1C1_fixed/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 5.56s, ave time: 0.40s

Making model...
Loading model from /data/yyang409/yancheng/SR_baseline_results/RCAN_HAN/x2/RCAN_HAN_x2_scratch_gammaL1C1_fixed/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 34.25s, ave time: 0.34s

Making model...
Loading model from /data/yyang409/yancheng/SR_baseline_results/RCAN_HAN/x2/RCAN_HAN_x2_scratch_gammaL1C1_fixed/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 105.43s, ave time: 1.05s

Making model...
Loading model from /data/yyang409/yancheng/SR_baseline_results/RCAN_HAN/x2/RCAN_HAN_x2_scratch_gammaL1C1_fixed/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 139.64s, ave time: 1.28s

